### PR TITLE
Update kernel and libvirt .yaml files with version information

### PR DESCRIPTION
**Pull request by fabianorosas**

### CODE REVIEW

**Comment by fabianorosas** [comment]
Related to https://github.com/open-power-host-os/builds/pull/96

**Comment by maurorodrigues** [comment]
Can one of the admins verify this patch?

File: kernel/kernel.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ Package:
  clone_url: 'https://github.com/open-power-host-os/linux.git'
  branch: 'hostos-devel'
  commit_id: 'a3a03f3'
+ version:
+   file: 'Makefile'
+   regex: 'VERSION.*([\w.-]+)\nPATCHLEVEL\s*=\s*(?P<patch>[\w.-]*)\nSUBLEVEL\s*=\s*(?(patch)([\w.-]*))'
```
Comment (fabianorosas) [inline kernel/kernel.yaml]: About '\d', the same answer from above applies.

The variables PATCHLEVEL and SUBLEVEL are optional in the kernel's 'kernelversion' make target:
https://github.com/open-power-host-os/linux/blob/powerkvm-v3.1.1/Makefile#L405

Not only that, but SUBLEVEL is only used if PATCHLEVEL is present.

File: libvirt/libvirt.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@ Package:
  clone_url: 'https://github.com/open-power-host-os/libvirt.git'
  branch: 'hostos-devel'
  commit_id: 'ddccbf6'
+ version:
+   file: 'configure.ac'
+   regex: 'AC_INIT\(\[[\w.-]+\],\s*\[([\w.-]+)\]'
```
Comment (olavphibm) [inline libvirt/libvirt.yaml]: My suggestion was actually to match anything inside the first [] ([^]] is negating the "]").
I've found some guidelines:
For package name: [\w-.+] https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines
For version: [^-/~\s] https://twiki.cern.ch/twiki/bin/view/Main/RPMAndDebVersioning

**Comment by maurorodrigues** [comment]
@fabianorosas could you please rebase this?

**Pull request merged**